app/src/main/java/com/example/android/letsparty/adapter/CircleTransform.java

```diff
@@ -0,0 +1,37 @@
+package com.example.android.letsparty.adapter;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import com.squareup.picasso.Transformation;
+
+public class CircleTransform implements Transformation {
+
+    @Override
+    public Bitmap transform(Bitmap source) {
+        int size = Math.min(source.getWidth(), source.getHeight());
+        int x = (source.getWidth() - size) / 2;
+        int y = (source.getHeight() - size) / 2;
+        Bitmap squaredBitmap = Bitmap.createBitmap(source, x, y, size, size);
+        if (squaredBitmap != source) {
+            source.recycle();
+        }
+        Bitmap bitmap = Bitmap.createBitmap(size, size, source.getConfig());
```

Comment:
It is creating a new image.. I am not sure what the image type is when we take the photo. If it is the bitmap, we can use this transform method.. But anyway, you can merge it to mater for now. These are things we can improve later.